test/templateParserTest.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var expect = require('chai').expect;
+var templateBuilder = require('../src/emailTemplateBuilder');
+
+var mockData = {
+    version: "0",
+    id: "ba1abec3-7aa3-1d0c-10b0-904559bf1f78",
+    'detail-type': "EC2 Instance State-change Notification",
+    source: "aws.ec2",
+    account: "some random value",
+    time: "2017-09-12T03:23:18Z",
+    region: "ap-southeast-2",
+    resources: ["arn:aws:ec2:ap-southeast-2:587392453450:instance/i-02eb595d1ea548b31"],
+    detail: {
+        'instance-id': "i-02eb595d1ea548b31",
+        state: "running"
+    }
+}
+
+var filename = '../templates/template.html';
+
+describe("Template Parser:", () => {
+
+    it('should open the html file', () => {
```

Comment:
I like the test. What other values can we obtain? I mean, to make it meaningful we need the account description / name and who created the instance

Reply:
Different Jira ticket.